das/storage_service.go

```diff
@@ -22,3 +23,11 @@ type StorageService interface {
 	ExpirationPolicy(ctx context.Context) arbstate.ExpirationPolicy
 	HealthCheck(ctx context.Context) error
 }
+
+func EncodeStorageServiceKey(b []byte) string {
+	return hexutil.Encode(b)
```

Comment:
`hexutil.Encode` includes a `0x` prefix in its output. Let's strip that off before returning.  (And compensate by prepending it before calling `hexutil.Decode` in the decode function.)

Reply:
Done.